### ISSUE TITLE
Firefox blocks Keys-0.2.3.msi download

### ISSUE THREAD

**Issue by beaugunderson** [keys-pub/app#48]
You may already be aware, but it looks like Firefox is currently blocking download of the .msi installer for 0.2.3 because it "could harm your device". Unsure if this is a signing issue or a false positive (VirusTotal says it's clean, fwiw).

**Comment by gabriel** [comment]
Yeah I think it's because I'm using a standard code signing and not an extended validation one?

**Comment by beaugunderson** [comment]
I wonder if it's a blanket block of MSI filetype? I wonder if it could be fixed just by zipping it...

**Comment by beaugunderson** [comment]
This came up in the context of onboarding someone on Windows so they could send me something  encrypted, so worth addressing I think :)

**Comment by gabriel** [comment]
The code signing cert was expired, maybe that was the issue? I just did a new build with a renewed cert, maybe that will help.
